### ISSUE TITLE
Missing config in new created traineddata

### ISSUE THREAD

**Issue by MPQC** [tesseract-ocr/tesstrain#310]
Hi. I'm trying to take the jpn_vert traineddata, and further train it with my own images. My command to run it looks like this:

```
make training MODEL_NAME=my-custom-model START_MODEL=jpn_vert TESSDATA=$TESSDATA
```

This works well, but if I run the following to create some traineddata while it's currently running:

```
make traineddata CHECKPOINT_FILES="$(ls -t data/my-custom-model/checkpoints/*.checkpoint | head -2)" MODEL_NAME=my-custom-model START_MODEL=jpn_vert TESSDATA=$TESSDATA
```

I would have expected it to use the same jpn_vert.config from the jpn_vert trainneddata and included it in the resulting model, but it doesn't have it. Is this expected?

**Comment by stweil** [comment]
Yes, it is currently implemented like that. There is no implementation which copies the configuration (and other components, for example the dictionary) from the original traineddata to the newly trained file(s).
